server/src/main/java/engine/ActiveState.java

```diff
@@ -31,22 +34,26 @@ public GameState run() throws IOException {
     Socket socket = playerRegistry.getCurrentPlayer().getSocket();
     ShotMessage shotMessage = messageReceiver.receiveShotMessage(socket);
     Fleet fleetUnderFire = playerRegistry.getFleetUnderFire();
+
     Shot shot = shotMessage.getShot();
+
     ShotResult result = round.fireShot(fleetUnderFire, shot);
     logShotInfo(shot, result);
-
+    String transcriptMessage = "Player: %s fired a shot: %d; it's a %s";
+    output.transcript(String.format(transcriptMessage, playerRegistry.getCurrentPlayerName(),
```

Comment:
Why did you extract variable transcriptMessage? For me it decrease a little bit readability